src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -172,14 +179,22 @@ onMounted(() => {
     } else {
         currentYear.value = previewYear.value
     }
-    tippyInstance = tippy(yearCursorInput.value, {
-        content: tooltipContent,
+    tippyYearOutsideRange = tippy(yearCursorInput.value, {
+        content: `${i18n.t('outside_valid_year_range')} ${ALL_YEARS[0]}-${ALL_YEARS[ALL_YEARS.length - 1]}`,
         arrow: true,
         hideOnClick: false,
         placement: 'bottom',
         trigger: 'manual',
         theme: 'danger',
     })
+    tippyTimeSliderInfo = tippy(timeSliderBar.value, {
```

Comment:
For this tippy maybe using the [followCursor](https://atomiks.github.io/tippyjs/v6/all-props/#followcursor) with `followCursor: 'horizontal',` might provide a better user experience ?